### DISCUSSION TITLE
Business oil peturelion and gas Nigeria Zamfara

### DISCUSSION THREAD

**Discussion by Aminu8199** [blockscout/blockscout#8618] [Ideas]
Assalamu alaikum sir after open mainnet countinio to get 2 coins everyday. Can I transactions with my coins to anybody or only covert my coins to others money I need can do

**Comment by Mojmir-R** [comment]
Hey, you are in the totally wrong place for such a question/request. I advise you to contact directly "Business motors car Nigeria Zamfara" if that is some entity.
